tests/test_core.py

```diff
@@ -838,8 +838,11 @@ def test_functions():
     imageio.mimsave(fname5, [im[:, :, 0] for im in ims])
     imageio.mimsave(fname5, ims)
     assert os.path.isfile(fname5)
-    
-    # Test volread()
+
+    if not sys.platform.startswith('linux'):
+        return
```

Comment:
Interesting, something must have been wrong with the `stent.npz` file in my `resources` directory.  I deleted it and the test passes, I'll remove this.